server/knexfile.js

```diff
@@ -31,4 +31,4 @@ const knexConfig = {
   ...knexSnakeCaseMappers({ underscoreBetweenUppercaseLetters: true })
 };
 
-export default knexConfig;
+export { knexConfig };
```

Comment:
this file should be with a default export
add exclude to the backend eslint file https://github.com/BinaryStudioAcademy/bsa-winter-2021-2022-bws/blob/development/backend/.eslintrc.yml#L11